### ISSUE TITLE
csv: string length overflow

### ISSUE THREAD

**Issue by moknomo** [tarantool/tarantool#2519]
Simple example with error
tarantool version 1.7.4-99-gb68732e
```
tarantool> csv = require('csv')
---
...

tarantool> csv.load('text ,')
---
- error: string length overflow
...
```

Crash is possible after several repetitions.
```PANIC: unprotected error in call to Lua API (string length overflow)```

**Comment by moknomo** [comment]
#bug

**Comment by Gerold103** [comment]
Can not reproduce after many many executions. Can you, please, update on the latest tarantool version and check, if the bug still exists?

**Comment by rtsisyk** [comment]
@moknomo ^^^

**Comment by moknomo** [comment]
Hmm, I can't reproduce it too.

**Comment by makorne** [comment]
```
tarantool
Tarantool 1.9.1-55-g1046f8516
type 'help' for interactive help
tarantool> csv = require('csv')
---
...

tarantool> csv.load('. ,')
---
- error: string length overflow
...
```

**Comment by kshcherbatov** [comment]
Already fixed on 2.1 by @Totktonada  with a9695b98b6a6fa5e8c130577f274f97819d30bf4 #3489

**Comment by kshcherbatov** [comment]
Fixed in the latest 1.10 release. Please upgrade.
